src/utils/daos.c

```diff
@@ -1016,8 +1016,63 @@ print_oclass_names_list(FILE *stream)
 	free(str);
 }
 
+#define FIRST_LEVEL_HELP() \
+do { \
+	fprintf(stream, \
+	"usage: daos RESOURCE COMMAND [OPTIONS]\n" \
+	"resources:\n" \
+	"	  pool             pool\n" \
+	"	  container (cont) container\n" \
+	"	  object (obj)     object\n" \
+	"	  version          print command version\n" \
+	"	  help             print this message and exit\n"); \
+	fprintf(stream, "\n"); \
+	fprintf(stream, "use 'daos help RESOURCE' for resource specifics\n"); \
+} while (0)
+
+#define ALL_CONT_CMDS_HELP() \
+do { \
+	fprintf(stream, "\n" \
+	"container (cont) commands:\n" \
+	"	  create           create a container\n" \
+	"	  destroy          destroy a container\n" \
+	"	  list-objects     list all objects in container\n" \
+	"	  list-obj\n" \
+	"	  query            query a container\n" \
+	"	  get-prop         get all container's properties\n" \
+	"	  set-prop         set container's properties\n" \
+	"	  get-acl          get a container's ACL\n" \
+	"	  overwrite-acl    replace a container's ACL\n" \
+	"	  update-acl       add/modify entries in a container's ACL\n" \
+	"	  delete-acl       delete an entry from a container's ACL\n" \
+	"	  set-owner        change the user and/or group that own a container\n" \
```

Comment:
The container ACL commands aren't in 0.9, nor is set-prop.

Reply:
Right and sorry for my lack of verification .... But should I just remove the missing commands from help or may be add code to report them as presently unimplemented ??

Reply:
In my view it would be best to remove them entirely.

Reply:
ok... BTW, get-acl is implemented! Will do in next commit.

Reply:
Huh, I'm surprised get-acl is in there. The enforcement of container ACLs isn't implemented in 0.9, so I'd be inclined to hide get-acl from the user, to avoid confusion. Just a personal preference though.

Reply:
> Huh, I'm surprised get-acl is in there. The enforcement of container ACLs isn't implemented in 0.9, so I'd be inclined to hide get-acl from the user, to avoid confusion. Just a personal preference though.

Well seems that your master PR-1783 which implemented get-acl had landed before the fork ... On my side, and after I had to make this lengthy clean/sync-up in both daos command on-line help and man-page to match what is "really" implemented/available in release/0.9 branch..., I would prefer to keep since it is in and to avoid to forget to put it back, just in case ...